tests/periph_cpuid/app.config

```diff
@@ -0,0 +1,2 @@
+CONFIG_MOD_PERIPH_CPUID=y
+CONFIG_MOD_TEST_UTILS_INTERACTIVE_SYNC=y
```

Comment:
Hmmm, can't we use the default test configuration here? With Make we could define a symbol `HAS_AUTOMATIC_TEST` (false by default) and based on that symbol set the default value of `CONFIG_MOD_TEST_UTILS_INTERACTIVE_SYNC` to yes, which can be overriden by individual applications. Does that make sense?

Reply:
> With Make we could define a symbol `HAS_AUTOMATIC_TEST` (false by default)

Do you refer to a Kconfig symbol here? How would we change the value of it from Make?

Reply:
Because currently in make we are doing some magic to enable `MODULE_TEST_UTILS_INTERACTIVE_SYNC` only when there is a pyton test script.